packages/router/src/router_module.ts

```diff
@@ -411,15 +411,16 @@ export interface ExtraOptions {
    * ];
    * ```
    *
-   * From the `ContainerComponent`, this will not work:
+   * From the `ContainerComponent`, this will not work with `relativeLinkResolution: 'legacy'`:
```

Comment:
```suggestion
   * From the `ContainerComponent`, you should be able to navigate to `AComponent` using the following `routerLink` but it will not work if `relativeLinkResolution` is set to `'legacy'`:
```
Small suggestion just to clarify what the expected behavior is.

Reply:
thanks @atscott, it's fixed now